### ISSUE TITLE
Could the "Edit Routine" modal remember its last position and size? 

### ISSUE THREAD

**Issue by DavidThomas48** [phpmyadmin/phpmyadmin#16243]
### Is your feature request related to a problem? Please describe.

I find I open and close the Routine editor regularly, and every time I do I have to reposition and resize it so I have the code section more or less filling the screen. This takes several mouse actions.

### Describe the solution you'd like

It would be very helpful if the modal dialog remembered its most recent position, size and internal layout.

### Describe alternatives you've considered
Just have to do it manually.

### Additional context

- phpMyAdmin version:

![image](https://user-images.githubusercontent.com/1977575/85972723-1608a900-ba14-11ea-88f7-c5405f18e35a.png)

**Comment by williamdes** [comment]
Hi
I totally understand the idea, still unsure where such a setting would be saved. Maybe in the local storage

**Comment by DavidThomas48** [comment]
Yes I agree – I use local storage for this sort of thing. It works fine.

**Comment by ibennetch** [comment]
This is a good enhancement idea.

I wonder if we could perhaps store this information in the phpMyAdmin table "pma__table_uiprefs" (specified by `$cfg['Servers'][$i]['table_uiprefs']`) and fall back on local storage if this table isn't configured.

**Comment by DavidThomas48** [comment]
May I just add to this request. It would be great if you could include the internal positioning of the modal (size of edit area, scroll positions, etc). Even after positioning and resizing the dialog, it can take many actions to get the code area to occupy the majority of the screen, especially if there are many parameters.
